### PR TITLE
Feature factor

### PR DESCRIPTION
**Pull request by timdmacdo**
This allows users to put percentage changes on the L/D, structural weights, and SFC to test the effects of technology factors.

### CODE REVIEW

File: trunk/SUAVE/Analyses/Aerodynamics/Markup.py [hunk 1]
```diff
@@ -45,8 +46,8 @@ def __defaults__(self):
         """           
         self.tag    = 'aerodynamics_markup'
         
-        self.geometry = Data()
```
Comment (WalterMaier) [inline trunk/SUAVE/Analyses/Aerodynamics/Markup.py]: what happened to geometry? not needed?

File: trunk/SUAVE/Components/Energy/Processes/Thrust.py [hunk 2]
```diff
@@ -180,7 +185,7 @@ def compute(self,conditions):
         Isp              = Fsp*a0*(1+bypass_ratio)/(f*g)
 
         #Computing the TSFC
-        TSFC             = 3600.*f*g/(Fsp*a0*(1+bypass_ratio))  
+        TSFC             = 3600.*f*g/(Fsp*a0*(1+bypass_ratio))*(1.-SFC_adjustment)
```
Comment (WalterMaier) [inline trunk/SUAVE/Components/Energy/Processes/Thrust.py]: Should not have 3600 in here, right?   should be secs-to-hour conversion or something...

File: trunk/SUAVE/Methods/Propulsion/turbofan_emission_index.py [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+
+# ----------------------------------------------------------------------
+#   Imports
+# ----------------------------------------------------------------------
+
+import numpy as np
+from SUAVE.Core import Data, Units
+
+# ----------------------------------------------------------------------
+#   turbofan_nox_emission_index
```
Comment (WalterMaier) [inline trunk/SUAVE/Methods/Propulsion/turbofan_emission_index.py]: name change.

File: trunk/SUAVE/Methods/Propulsion/turbofan_emission_index.py [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+    """
+    
+    results = turbofan(state)
+    p3      = turbofan.combustor.inputs.stagnation_pressure/Units.psi
+    T3      = turbofan.combustor.inputs.stagnation_temperature/Units.degR 
+    T4      = turbofan.combustor.outputs.stagnation_temperature/Units.degR
+    mdot    = state.conditions.weights.vehicle_mass_rate
+    I       = state.numerics.time.integrate
+    
+    NOx = .004194*T4*((p3/439.)**.37)*np.exp((T3-1471.)/345.)
```
Comment (WalterMaier) [inline trunk/SUAVE/Methods/Propulsion/turbofan_emission_index.py]: what are these?

File: trunk/SUAVE/Methods/Weights/Correlations/Tube_Wing/empty.py [hunk 5]
```diff
@@ -82,6 +83,12 @@ def empty(vehicle):
           t_c - thickness-to-chord ratio of the vertical tail                    [dimensionless]
           sweep - sweep angle of the vertical tail                               [radians]
           t_tail - factor to determine if aircraft has a t-tail, "yes"           [dimensionless]
+          
+      settings.weight_reduction_factors.
+          main_wing                                                              [dimensionless] (.1 is a 10% weight reduction)
+          empennage                                                              [dimensionless] (.1 is a 10% weight reduction)
```
Comment (WalterMaier) [inline trunk/SUAVE/Methods/Weights/Correlations/Tube_Wing/empty.py]: should dimensionless be [-]?

Comment (timdmacdo) [reply trunk/SUAVE/Methods/Weights/Correlations/Tube_Wing/empty.py]: That's what I usually use but we don't have a standard. I matched it to the rest of the file.

File: trunk/SUAVE/Methods/Propulsion/turbofan_emission_index.py (hunk 4 above)
Comment (timdmacdo) [reply trunk/SUAVE/Methods/Propulsion/turbofan_emission_index.py]: This is a fit and the units are visible in the doc string and the conversion factor when they are unpacked.

**Comment by planes** [comment]
You can merge this if you'd like.

**Pull request merged**